ngrok/plan.sh

```diff
@@ -0,0 +1,77 @@
+pkg_name=ngrok
+pkg_origin=core
+pkg_version=_set_from_downloaded_zip_file_
+pkg_description="Expose local servers behind NATs and firewalls to the public internet over secure tunnels."
+pkg_upstream_url=https://ngrok.com/
+pkg_license=('proprietary')
```

Comment:
I did some investigation about this.

ngrok has two major versions. Version 1, which is available at the Github repository that @eeyun linked, is Apache 2.0 licensed. However, it appears that the current version, 2.0, and the version downloaded via the `pkg_source` in this plan is a proprietary license. Per the [Terms of Service](https://ngrok.com/tos), it does not appear that we can distribute a package of ngrok 2.x+.

> You are granted a limited, non-sublicensable license to access and use the Site and the ngrok Site Materials solely in connection with the Services. Such license is subject to this Agreement and does not include: (a) any resale or commercial use of the Site or the Site Materials therein; (b) the distribution, public performance or public display of any Site Materials except in connection with your authorized use of the Site and the Services; (c) modifying or otherwise making any derivative uses of the Site and/or the Site Materials, or any portion thereof

Further, it is likely that ngrok 1.x has stability issues per the [README.md](https://github.com/inconshreveable/ngrok/blob/master/README.md) and should not be used in "production." As the use case seems to be primarily for exposing services running on a development system, [that's probably fine](https://twitter.com/oising/status/298464920717099009). I think if we're going to create an ngrok package, it should be based on 1.x, and be clear about intent.